test/test_optim.py

```diff
@@ -394,6 +410,11 @@ def test_peak_memory_foreach(self, device, dtype, optim_info):
                     # bias_correction, mus, and mu_nexts
                     nintermediates = 5
 
+                if optim_cls.__name__ == "RAdam":
+                    # RAdam has six intermediates with capturable
```

Comment:
is this the min we could go

Reply:
Lowered to 4 after @vadimkantorov 's suggestions!